pkg/cloud/services/networking/securitygroups.go

```diff
@@ -147,21 +153,35 @@ func generateControlPlaneGroup(clusterName string) infrav1.SecurityGroup {
 					RemoteIPPrefix: "0.0.0.0/0",
 				},
 				{
-					Direction:      "ingress",
-					EtherType:      "IPv4",
-					PortRangeMin:   22,
-					PortRangeMax:   22,
```

Comment:
maybe I missed something ,does this means not able to ssh anymore?

Reply:
We can not use ssh access anymore. Because I really implemented [Check required ports](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#check-required-ports).

Reply:
I am ok with it ,however, for debug, without ssh it will be really not easy
we at least need provide a way to enable ssh

or at worst case, create a doc guide user how to enable it by modifying the code...

Reply:
We can open ssh port by changing security groups dynamically after VM has been booted if some problems occurred( Actually I was doing so 😅 ).
I totally agree adding documentation.

Reply:
It is much easier to add existing security group to your yaml than modifying the code like the following.

```yaml
      securityGroups:
      - name: k8s-cluster-default-capi-quickstart-secgroup-controlplane
      - name: default
```
this default security group enables ssh port.

Reply:
@jichenjc Updated docs.
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/555/files#diff-76e731333fb756df3bff5ddb3b731c46

Reply:
thanks for the update